src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -247,6 +254,20 @@ public static HelpCommand chooseHelpCommand(String helpCommandName) throws Illeg
             return new FindHelpCommand();
         case "date":
             return new DateHelpCommand();
+        case "listmod":
+            return new ListModuleHelpCommand();
+        case "help":
+            return new SpecifiedAidHelpCommand();
+        case "bye":
+            return new ExitHelpCommand();
+        case "week":
+            return new WeekHelpCommand();
+        case "delmod":
+            return new DeleteModHelpCommand();
+        case "showmod":
+            return new ShowModHelpCommand();
+        case "addmod":
+            return new AddModHelpCommand();
```

Comment:
Good use of case statements.

Reply:
Perhaps there is no need for the case for help? The parent helpcommand does have a execute function that prints out the full help message